Assets/MRTK/SDK/Experimental/Drawstring/Scripts/TwoHandleManipulator.cs

```diff
@@ -0,0 +1,937 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Physics.Experimental;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.Physics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.Serialization;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.Toolkit.UI.Experimental
+{
+    /// <summary>
+    /// TODO insert drawstring summary
+    /// </summary>
+    [HelpURL("https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/README_ObjectManipulator.html")]
+    public class TwoHandleManipulator : MonoBehaviour, IMixedRealityPointerHandler, IMixedRealityFocusChangedHandler
+    {
+        #region Public Enums
+
+        /// <summary>
+        /// Specifies the stretch behavior when grabbing with only one handle.
+        /// 
+        /// StretchAboutCenter will make one-handed stretching essentially identical
```

Comment:
Great comments - note that it would be better to have the comments about each property actually be doccomments on each property (so someone who is looking at autocomplete for KeepOtherEndSteady will see documentation when doing that, and so that when they read the auto-gened docs on github.io they see the comments on the property itself)

I think I've seen examples of us doing this incorrectly elsewhere in our code but we should strive to do the right thing here.